ingest/a_vs_b_de.py

```diff
@@ -0,0 +1,65 @@
+#using https://nbviewer.org/github/theislab/diffxpy_tutorials/blob/master/diffxpy_tutorials/test/multiple_tests_per_gene.ipynb
+import anndata
+import matplotlib.pyplot as plt
+import seaborn as sns
```

Comment:
Blocker: Please add this to `requirements.txt`, so we can install it via `pip install -r requirements.txt`.

In particular, this would be best to add under the ["Dev dependencies" section](https://github.com/broadinstitute/scp-ingest-pipeline/blob/development/requirements.txt#L17), for easier reasoning about its purpose here.  My understanding is Seaborn would only be used for local debugging, e.g. making volcano plots that SCP Ingest Pipeline contributors could use for quick local checks.